R/makeMBOLearner.R

```diff
@@ -34,21 +42,26 @@
 #'   Further parameters passed to the constructed learner.
 #'   Will overwrite mlrMBO's defaults.
 #' @return [\code{Learner}]
+#' @aliases mbo_default_learner
 #' @export
-makeMboLearner = function(control, fun, ...) {
+makeMBOLearner = function(control, fun, ...) {
   assertClass(control, "MBOControl")
   assertClass(fun, "smoof_function")
 
   ps = getParamSet(fun)
   if (isNumeric(ps, include.int = TRUE)) {
-    lrn = makeLearner("regr.km", predict.type = "se", covtype = "matern5_2", optim.method = "gen")
+    lrn = makeLearner("regr.km", predict.type = "se", covtype = "matern3_2", optim.method = "gen")
     if (!isNoisy(fun))
       lrn = setHyperPars(lrn, nugget.stability = 10^-8)
     else
       lrn = setHyperPars(lrn, nugget.estim = TRUE, jitter = TRUE)
   } else {
-    lrn = makeLearner("regr.randomForest", predict.type = "se", se.method = "sd")
+    lrn = makeLearner("regr.randomForest", predict.type = "se", se.method = "jackknife", keep.inbag = TRUE)
   }
+  
+  if (hasRequires(ps))
+    lrn = makeImputeWrapper(lrn, classes = list(numeric = imputeMax(2), factor = imputeConstant("__miss__")))
```

Comment:
If we have a numerical parameter space we want to combine imputeMax with Kriging? Is that a good idea?

I know that this case is really rare but still... for me this belongs just to Tree Based Learners.